### PR TITLE
Removed default props as it is no more supported

### PR DESCRIPTION
**Pull request by shruti-14**

### CODE REVIEW

File: src/lib/components/DefinitionTooltip/DefinitionTooltip.js [hunk 1]
```diff
@@ -30,24 +30,24 @@ import positionTooltip from "../../helpers/positionTooltip.js";
...
     definition,
     explanation,
     id,
     link,
     onClick,
-    position,
+    position = "top",
     term,
-    url
+    url,
```
Comment (bgetting) [inline src/lib/components/DefinitionTooltip/DefinitionTooltip.js:43]: Do we need this trailing comma?

File: src/lib/components/DefinitionTooltip/DefinitionTooltip.js [hunk 2]
```diff
@@ -68,7 +68,7 @@ const DefinitionTooltip = props => {
         aria-labelledby={tooltipElementId}
         className={triggerClass}
         dangerouslySetInnerHTML={{
-          __html: term
+          __html: term,
```
Comment (bgetting) [inline src/lib/components/DefinitionTooltip/DefinitionTooltip.js:71]: Are these trailing commas being added by a linter?

**Review by bgetting — APPROVED** [review]
Looks good, but I'm wondering about all the trailing commas.

File: src/lib/components/DefinitionTooltip/DefinitionTooltip.js (hunk 1 above)
Comment (shruti-14) [reply src/lib/components/DefinitionTooltip/DefinitionTooltip.js:43]: It is added by code formatter.

File: src/lib/components/DefinitionTooltip/DefinitionTooltip.js (hunk 2 above)
Comment (shruti-14) [reply src/lib/components/DefinitionTooltip/DefinitionTooltip.js:71]: yes

**Pull request merged**